src/wallet/wallet.ts

```diff
@@ -265,22 +266,19 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
 
     const handleCreate = async (data: WalletStatus) => {
       this.walletId = data.walletId;
+
       if (data.status === 'creating') {
-        this.walletStatusInterval = setInterval(async () => {
```

Comment:
Should we remove this attribute from the class and constructor?

Reply:
Yes we should, thanks

Done in 82f4de533c16baad51c83356a3609c393b5175aa